Farsi/admin_forums.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+// Language definitions used in admin_forums.php
+$lang_admin_forums = array(
+
+'Forum added redirect'		=>	'انجمن اضافه شد. درحال تغییر مسیر …',
+'Forum deleted redirect'	=>	'انجمن حذف شد. درحال تغییر مسیر …',
+'Forums updated redirect'	=>	'انجمن ها بروزرسانی شدند. درحال تغییر مسیر …',
+'Forum updated redirect'	=>	'انجمن بروزرسانی شد. درحال تغییر مسیر …',
+'Perms reverted redirect'	=>	'مجوزها به حالت پیش فرض برگشتند. درحال تغییر مسیر …',
+'Must enter name message'	=>	'شما باید برای انجمن یک نامی را وارد کنید.',
+'Must be integer message'	=>	'موقعیت باید یک مقدار صحیح مثبت باشد.',
+'New forum'					=>	'انجمن جدید',
+
+// Entry page
+'Add forum head'			=>	'افزودن انجمن',
+'Create new subhead'		=>	'ایجاد یک انجمن جدید',
+'Add forum label'			=>	'افزودن انجمن به مجموعه',
+'Add forum help'			=>	'مجموعه ای را که مایل هستید انجمن جدید را به آن اضافه کرده را انتخاب کنید.',
+'Add forum'					=>	'افزودن انجمن',
+'No categories exist'		=>	'هیچ مجموعه ای وجود ندارد',
+'Edit forums head'			=>	'ویرایش انجمن',
+'Category subhead'			=>	'مجموعه:',
+'Forum label'				=>	'انجمن',
+'Edit link'					=>	'ویرایش',
+'Delete link'				=>	'حذف',
+'Position label'			=>	'موقعیت',
+'Update positions'			=>	'بروزرسانی موقعیت ها',
+'Confirm delete head'		=>	'تائید حذف انجمن',
+'Confirm delete subhead'	=>	'بسیار مهم ! قبل از حذف مطالعه شود',
+'Confirm delete info'		=>	'آیا مطمئن هستید که میخواهید انجمن <strong>%s</strong> را حذف کنید؟',
+'Confirm delete warn'		=>	'<strong>هشدار!</strong> با حذف یک انجمن تمام ارسال ها (درصورت وجود) در این انجمن نیز حذف خواهند شد!',
+
+// Detailed edit page
+'Edit forum head'			=>	'ویرایش انجمن',
+'Edit details subhead'		=>	'ویرایش جزئیات انجمن',
+'Forum name label'			=>	'نام انجمن',
+'Forum description label'	=>	'توضیحات (HTML)',
```

Comment:
Yes , i think this is the best way , coz something can't to be translated , especially in such thing .